package.json

```diff
@@ -16,7 +16,9 @@
     "test": "jest -u",
     "seeder": "node database/seed-data/seeder.js",
     "build-dev": "webpack --config webpack.config.js --watch",
-    "server-dev": "nodemon server/server.js"
+    "build": "webpack --config webpack.config.js",
```

Comment:
I am unsure of how your webpack config file looks. Just a heads up to set the mode option in the config file to production instead of development for deployment purposes. It should help bring down bundle size along with a few other optimizations.

Reply:
Good tip, thank you!